### PR TITLE
address missed impacket dependency check

### PR DESCRIPTION
**Pull request by jmartin-tech**
Fix module cache load for `ms17_010_eternalblue_win8` due to missing dependency check

## Verification

List the steps needed to make sure this thing works

- [x] Start `msfconsole`
- [x] check `~/.msf4/logs/framework.log` for error loading `ms17_010_eternalblue_win8`
- [x] **Verify** no errors found in latest startup

### CODE REVIEW

File: lib/msf/core/modules/external/templates/remote_exploit_cmd_stager.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@ require 'msf/core/modules/external/bridge'
 require 'msf/core/module/external'
 
 class MetasploitModule < Msf::Exploit::Remote
-  Rank = <% meta[:rank] %>
```
Comment (wvu-r7) [inline lib/msf/core/modules/external/templates/remote_exploit_cmd_stager.erb]: Oof!

File: modules/exploits/windows/smb/ms17_010_eternalblue_win8.py [hunk 2]
```diff
@@ -54,7 +54,7 @@
         - The idea is from "Bypassing Windows 10 kernel ASLR (remote) by Stefan Le Berre" (see link in reference)
         - The exploit is also the same but we need to trigger bug twice
         - First trigger, set MDL.MappedSystemVa to target pte address
-          - Write '\x00' to disable the NX flag
```
Comment (wvu-r7) [inline modules/exploits/windows/smb/ms17_010_eternalblue_win8.py]: Looks like this was copied directly from the original docstring, but the fact that it was added to `metadata` sent it over JSON-RPC.

**Comment by wvu-r7** [comment]
Great catches!!

**Review by wvu — APPROVED** [review]

File: modules/exploits/windows/smb/ms17_010_eternalblue_win8.py (hunk 2 above)
Comment (wvu-r7) [reply modules/exploits/windows/smb/ms17_010_eternalblue_win8.py]: Or, rather, it sent nothing (null).

**Pull request merged**